integrations/fastapi/playground/requirements.txt

```diff
@@ -11,7 +11,7 @@ python-dotenv==1.0.1
 PyYAML==6.0.1
 scalar-fastapi==1.0.1
 sniffio==1.3.1
-starlette==0.40.0
+starlette==0.37.2
```

Comment:
why that? 🤔

Reply:
this matches the current version used by the integration

Reply:
ah, ok 👍